File: EmbyStat.Web/ClientApp/src/app/movie/models/movieStats.ts
import { Card } from '../../shared/models/card';
import { TimespanCard } from '../../shared/models/timespanCard';
import { MoviePoster } from '../../shared/models/moviePoster';

export class MovieStats {
  public movieCount: Card ;
  public genreCount: Card ;
  public boxsetCount: Card ;
  public mostUsedContainer: Card ;
  public highestRatedMovie: MoviePoster ;
  public lowestRatedMovie: Card ;
  public longestMovie: Card ;
  public shortestMovie: Card ;
  public oldestPremieredMovie: Card ;
  public youngestPremieredMovie: Card ;
  public youngestAddedMovie: Card ;
  public mostFeaturedMovieActor: Card ;
  public mostFeaturedMovieDirector: Card ;
  public lastPlayedMovie: Card ;
  public totalPlayableTime: TimespanCard ;
}

File: EmbyStat.Services/Models/Movie/MovieStats.cs
using EmbyStat.Services.Models.Stat;

namespace EmbyStat.Services.Models.Movie
{
    public class MovieStats
    {
        public Card MovieCount { get; set; }
        public Card GenreCount { get; set; }
        public Card BoxsetCount { get; set; }
        public Card MostUsedContainer { get; set; }
        public MoviePoster HighestRatedMovie { get; set; }
        public MoviePoster LowestRatedMovie { get; set; }
        public MoviePoster LongestMovie { get; set; }
        public MoviePoster ShortestMovie { get; set; }
        public MoviePoster OldestPremieredMovie { get; set; }
        public MoviePoster YoungestPremieredMovie { get; set; }
        public MoviePoster YoungestAddedMovie { get; set; }
        public Card MostFeaturedMovieActor { get; set; }
        public Card MostFeaturedMovieDirector { get; set; }
        public Card LastPlayedMovie { get; set; }
        public TimeSpanCard TotalPlayableTime { get; set; }
    }
}
